client/state/domains/nameservers/actions.js

```diff
@@ -1,42 +0,0 @@
-/**
- * Internal dependencies
- */
-import {
-	DOMAIN_NAMESERVERS_FETCH,
```

Comment:
Those action types are now unused and can be removed from the `action-types.ts` file.

Reply:
fixed with https://github.com/Automattic/wp-calypso/pull/53657/commits/ce5ac2aa0ed11079af18e36fe8ad164e4aa1a945